src/actions/posts.js

```diff
@@ -888,6 +890,93 @@ export function getOpenGraphMetadata(url) {
     };
 }
 
+export function getFlaggedPosts() {
+    return async (dispatch, getState) => {
+        const state = getState();
+        const userId = getCurrentUserId(state);
+        const teamId = getCurrentTeamId(state);
+
+        dispatch({type: SearchTypes.SEARCH_POSTS_REQUEST});
+        try {
+            const data = await Client4.getFlaggedPosts(userId, '', teamId);
+            dispatch(batchActions([
+                {type: SearchTypes.SEARCH_POSTS_SUCCESS},
+                {
+                    type: SearchTypes.RECEIVED_SEARCH_TERM,
+                    data: {
+                        term: null,
+                        do_search: false,
+                        is_mention_search: false
+                    }
+                },
+                {
+                    data,
+                    type: SearchTypes.RECEIVED_FLAGGED_POSTS
+                },
+                {
+                    data,
+                    type: SearchTypes.RECEIVED_SEARCH_POSTS
```

Comment:
I don't think we need this `RECEIVED_SEARCH_POSTS` action dispatched here, since we want to separate the search results for posts and the getting flagged posts